DayrSuryan/DSEthiop7.xml

```diff
@@ -286,12 +294,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <supportDesc>
                         <support>
                            <material key="paper"/>
-                           <p>The manuscript is made of an industrially produced Italian notebook
-                              (the title <foreign xml:lang="it">“Quaderno”</foreign>
-                              appears at the end, on <locus target="#iiv"/>).
-                              The notebook has been turned around for writing,
-                              the title <foreign xml:lang="it">“Quaderno”</foreign> now appears
-                              upside down. </p>
+                           <p>The manuscript is a re-utilized industrially produced Italian notebook.
```

Comment:
reutilised?

Reply:
ok. strange, New Oxford Style has re-embark, re-emerge, re-equip, re-establish. Perhaps we consider reutilise not just "utilise the second time", but "reconsider as smth new".

Reply:
it might be right. but let's keep the s, re-utilised